public/404.html

```diff
@@ -1,3 +1,5 @@
+<!-- This page is obsolete and has been replaced by error_404 -->
```

Comment:
nit: can these files be deleted (`public/404.html`, `public/500.html`)?

Reply:
`500.html` can be safely deleted
`404.html` can be deleted, but unless `config.exceptions_app = self.routes` is added, users will only see a blank page

Reply:
In the interest of minimizing dead code, I'll delete the files and bring attention to the change required with the next release. Besides, seeing a broken error page isn't much better than a blank page.